MediaBrowser.MediaEncoding/Attachments/AttachmentExtractor.cs

```diff
@@ -134,7 +135,7 @@ await ExtractAllAttachmentsInternal(
 
                     if (Directory.Exists(outputPath))
                     {
-                        File.Create(Path.Join(outputPath, id));
+                        await File.Create(outputIdPath).DisposeAsync().ConfigureAwait(false);
```

Comment:
I think you can just close the file opened.

Reply:
Probably, but the [documentation ](https://learn.microsoft.com/en-us/dotnet/api/system.io.filestream?view=net-8.0#methods) suggests that disposing is preferred over `Close`. Switching to a plain `Dispose` yields a [CA1849](https://learn.microsoft.com/en-ca/dotnet/fundamentals/code-analysis/quality-rules/ca1849) warning.